### ISSUE TITLE
Replace custom configuration aliases

### ISSUE THREAD

**Issue by TheAssassin** [mailcow/mailcow-ansiblerole#52]
It would be a lot handier to just use the keys from `mailcow.conf` directly as variables. This would also lower the maintenance burden, as you no longer need to keep the configuration settings of this role in sync with the upstream project.

Related to #51 and #39.

**Comment by DerLinkman** [comment]
Yeah, feel free to open up a PR for it if you like. Sounds good in my opinion.
